pkg/operator/etcdcertsigner/etcdcertsignercontroller.go

```diff
@@ -126,17 +134,24 @@ func NewEtcdCertSignerController(
 		metricsClientCert:     metricsClientCert,
 	}
 
+	signerExpirationGauge := metrics.NewGaugeVec(&metrics.GaugeOpts{
+		Name: signerExpirationMetricName,
+		Help: "Report observed days to expiration (NOT_AFTER) of a given signer certificate over time",
```

Comment:
Just to clarify what NOT_AFTER means here, i.e not after expiration? Or is this an env var of sorts?

Reply:
it's certificate lingo: https://serverfault.com/a/339471 - but I think we can just leave this out of the help text